roles/afm_cos_configure/tasks/afm_sensor_configure.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: configure | Initialize lists of AFM Nodes
+  set_fact:
+   scale_gateway_nodes: []
+
+- name: cluster | Find AFM gateway nodes
+  add_host:
+    name: "{{ item }}"
+    groups: scale_gateway_nodes
+  when: hostvars[item].scale_cluster_gateway | bool
+  with_items: "{{ ansible_play_hosts }}"
+  changed_when: false
+
+- name: configure | Setting AFM Sensors on AFM gateway nodes
+  vars:
+     scale_gateway_node: "{{ groups['scale_gateway_nodes'] | list }}"
+  command: "{{ scale_command_path }}mmperfmon config update GPFSAFMFS.restrict={{ scale_gateway_node | join(',') }} GPFSAFMFS.period=10 GPFSAFMFSET.restrict={{ scale_gateway_node | join(',') }} GPFSAFMFS.period=10 GPFSAFMFSET.restrict={{ scale_gateway_node | join(',') }} GPFSAFMFSET.period=10 GPFSAFM.restrict={{ scale_gateway_node | join(',') }} GPFSAFM.period=10"
+  when: groups['scale_gateway_nodes'] | list | length > 0
+  run_once: true
```

Comment:
We need to add check if permon sensor file is exist then only it should be executed else it will fail.

` - " 'mmperfmon: There is no performance monitoring configuration data.' not in scale_zimon_conf_sensors.stderr"`

Reply:
@rajan-mis I have created an issue.
https://github.com/IBM/ibm-spectrum-scale-install-infra/issues/812